concurrent/actor/EventLoopClock.java

```diff
@@ -15,21 +15,21 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-package grakn.common.concurrent.actor.eventloop;
+package grakn.common.concurrent.actor;
 
 import java.util.Random;
 import java.util.concurrent.ThreadLocalRandom;
 import java.util.function.Supplier;
 
-public class GlobalSystem {
+public class EventLoopClock {
```

Comment:
This is not just time, but also random.

Reply:
Yeah you're right. Therefore I've renamed it to `System` (it will be available as `EventLoop.System`)